service/src/main/kotlin/no/nav/su/se/bakover/service/revurdering/RevurderingServiceImpl.kt

```diff
@@ -1241,6 +1241,7 @@ internal class RevurderingServiceImpl(
                                         )
                                         throw UtbetalingFeiletException(feil)
                                     }
+                                    vedtak
```

Comment:
*Kommentar gjelder for alle endringer i denne PR*
Har vi none asserts på rekkefølgen her? Her er det vel veldig viktig at det ikke skjer noe som kan tryne etter utbetalingen har gått? Hvis vi ikke har noe sånt foreslår jeg at vi kan spandere på oss en `inOrder + verify` selv om dette kanskje er i grenseland til å være persona non grata. Burde vi spandere på oss en kommentar ellerno og?

Reply:
@simejo husker nok, men mener det er en inOrder-verify-test på "happy pathen" her. Når det er sagt, så tror jeg ikke transaksjonen funker sånn som vi ønsker her, ref. https://trello.com/c/j4pFwOED/1148-transaksjoner-og-left En left vil vel i praksis bare ignoreres, og verken rulle tilbake det foregående eller stoppe det neste.